server/cursor.js

```diff
@@ -0,0 +1,42 @@
+import {check, Match} from 'meteor/check';
+import {Meteor} from 'meteor/meteor';
+
+import {Cursor} from '/lib/cursor';
+import {User} from '/lib/user';
+
+Meteor.methods({
+  'Cursor.update'(args) {
+    check(args, {
+      contentKey: Match.DocumentId,
+      position: Match.Integer,
```

Comment:
BTW, cursors can also be selections. :-) See the issue description for more details (on GitHub).

Reply:
Oh! I didn't notice that. Should we render other users selection areas as well? Or just the position where their selections end?

Reply:
I mean, we should render all selections by all users. For current user ProseMirror is already rendering it, so we should just render non-current user's selections ourselves.